data/src/main/java/tech/pegasys/artemis/data/TimeSeriesRecord.java

```diff
@@ -248,4 +250,41 @@ public int hashCode() {
         finalizedBlockRoot,
         finalizedStateRoot);
   }
+
+  @Override
+  public String toJSON() {
+    Gson gson = new GsonBuilder().create();
```

Comment:
It's better if the TimeSeriesRecord is completely agnostic to the output format.  It would be better to keep the toCSV and toJSON code in their respective Provider classes under .toString()